clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -254,7 +254,7 @@ message Query {
 message SearchRequest {
     string indexName = 1; //index to search against
     int32 startHit = 2; // Which hit to start from (for pagination); default: 0
-    int32 topHits = 3; // How many top hits to retrieve; default: 10
+    int32 topHits = 3; // How many top hits to retrieve; default: 10. For pagination: set it to startHit + window_size. 
```

Comment:
I think you might want to rebuild the swagger spec to account for the updated comment? It will show up in swagger_ui

Reply:
++ Regenerated grpcGateway (is it correct way to update the swagger schema)?